testnet/testnet-deploy-l2-contracts.sh

```diff
@@ -87,6 +115,7 @@ echo "Deploying contracts to Layer 2 using obscuro hardhat container..."
 docker run --name=hh-l2-deployer \
     --network=node_network \
     -e NETWORK_JSON="${network_cfg}" \
+    -v deploymentsvol:/home/go-obscuro/contracts/deployments \
```

Comment:
Do want this mount here ?

Reply:
Yep, to access the L1 deployments. Is technically like mounting files.